assets/styles/styles.css

```diff
@@ -0,0 +1,308 @@
+          border: 0;
+          background-color: #ffa62b;
+          font-size: 1.4rem;
+          font-weight: 700;
+          color: #ffffff;
+        }
+        .hero__btn2 {
+          margin: 0;
+          padding: 0;
+          width: 15.4rem;
+          border-radius: 3.7rem;
+          border: 0.1rem solid #ffffff;
+          background: transparent;
+          font-size: 1.4rem;
+          font-weight: 700;
+          color: #ffffff;
+        }
+      }
+    }
+  }
+}
+
+.footer {
+  display: flex;
+  justify-content: center;
+
+  .container {
+    max-width: 144rem;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .footer__top {
+    max-width: 105rem;
+    width: 100%;
+    height: 13.8rem;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    align-items: center;
+
+    .footer__logo {
+      margin-right: auto;
+      text-decoration: none;
+      color: white;
+      font-size: 2.4rem;
+      font-weight: 700;
+    }
+
+    .footer__socials {
+      all: unset;
+      list-style: none;
+      display: flex;
+      gap: 2rem;
+      margin: 0;
+      padding: 0;
+      padding-right: 4rem;
+      height: 2.4rem;
+
+      .socials__link {
+        text-decoration: none;
+
+        .link__img {
+          width: 2.4rem;
+          height: 2.4rem;
+        }
+      }
+    }
+  }
+  .footer__main {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 3rem;
+    width: 105rem;
+    height: 31rem;
+    padding: 0;
+    margin: 0;
+
+    .main__item {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      gap: 2rem;
+      width: 14.8rem;
+
+      .item__title {
+        font-size: 1.6rem;
+        font-weight: 700;
+        padding: 0;
+        margin: 0;
+        height: 2.4rem;
+      }
+
+      .item__list {
+        font-size: 1.4rem;
+        font-weight: 600;
+        padding: 0;
+        margin: 0;
+        display: flex;
+        gap: 1rem;
+        flex-direction: column;
+        list-style: none;
+
+        .list__link {
+          height: 2.4rem;
+
+          a {
+            color: #bdbdbd;
+          }
+        }
+      }
+      .item__form {
+        width: 31.9rem;
+        height: 5.6rem;
+        display: flex;
+        border: 0.1rem solid #e6e6e6;
+        border-radius: 0.5rem;
+
+        .form__input {
+          width: 18.5rem;
+          height: 5.6rem;
+          /*border: 0;*/
+          padding: 0;
+          font-size: 1.4rem;
+          font-weight: 400;
+          color: #bdbdbd;
+          padding-left: 2rem;
+          border: 0.1rem;
+        }
+        .form__button {
+          width: 12rem;
+          height: 5.8rem;
+          padding: 0;
+          background-color: #ffa62b;
+          font-size: 1.4rem;
+          font-weight: 400;
+          text-align: center;
+          border: 0.1rem;
```

Comment:
it is not necessary to use rem everywhere. 
Use it only for parameters that might change in case you change the base font size. It's more about responsiveness. I don't think you'd like to make the border width 0.5px if you make the base fontsixe 5p on small screens. 

So in this case better use fixed "1px" unit so border would be the same for all devices